sky/skylet/providers/gcp/constants.py

```diff
@@ -9,6 +9,26 @@
     "mtu": 1460,
     "routingConfig": {"routingMode": "GLOBAL"},
 }
+# Required firewall rules for SkyPilot to work.
+FIREWALL_RULES_REQUIRED = [
+    {
+        "direction": "INGRESS",
+        "allowed": [
+            {'IPProtocol': 'tcp', 'ports': ['0-65535']},
+            {'IPProtocol': 'udp', 'ports': ['0-65535']},
+        ],
+        "sourceRanges": ["10.128.0.0/9"],
+    },
+    {
+        "direction": "INGRESS",
+        "allowed": [{
+            "IPProtocol": "tcp",
+            "ports": ["22"],
+        }],
+        "sourceRanges": ["0.0.0.0/0"],
+    }
+]
```

Comment:
Add a comment for this rule, such as `Allow ssh connection`.

Reply:
Added!